src/draw/handler/Draw.Feature.js

```diff
@@ -13,13 +13,14 @@ L.Draw.Feature = L.Handler.extend({
 		if (options && options.shapeOptions) {
 			options.shapeOptions = L.Util.extend({}, this.options.shapeOptions, options.shapeOptions);
 		}
-		L.setOptions(this, options);
+
+		L.ToolbarAction.prototype.initialize.call(this, options);
```

Comment:
Sorry maybe i should have said this will make the draw handlers require that Leaflet.toolbar is also included even if the features are not used.